src/components/common/Radio/radio.test.tsx

```diff
@@ -3,8 +3,14 @@ import { render, fireEvent } from 'test-utils';
 
 import { Basic } from './radio.stories';
```

Comment:
creating a seperate issue to remove the stories from the tests. We should not test the storybook implementation but the actual component.